collect_app/src/main/java/org/odk/collect/android/fragments/AppListFragment.java

```diff
@@ -60,7 +62,8 @@
     protected SimpleCursorAdapter mListAdapter;
     protected LinkedHashSet<Long> mSelectedInstances = new LinkedHashSet<>();
     protected EditText mInputSearch;
-
+    private int moldSelected;
```

Comment:
I didn’t study this carefully, but is it necessary to have duplicate code here and in AppListActivity?